packages/perseus/src/widgets/label-image.tsx

```diff
@@ -644,12 +645,21 @@ class LabelImage extends React.Component<LabelImageProps, LabelImageState> {
                 mediaQueries.xsOrSmaller.replace("@media ", ""),
             ).matches;
 
-            let side;
+            let side: "bottom" | "left" | "right" | "top";
+            let markerPosition;
```

Comment:
Previously, we calculated the position of the marker relative to the popup but didn't write it to a variable. Storing now to use in dynamic CSS.